relay/sources/rudder-package/src/archive.rs

```diff
@@ -90,7 +88,11 @@ impl Rpkg {
     fn get_txz_dst(&self, txz_name: &str) -> String {
```

Comment:
we use the result as a path so returning a PathBuf would make more sense (and make the return less awkward)